crates/bindings-csharp/BSATN.Runtime/BSATN.Runtime.csproj

```
@@ -26,13 +26,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" OutputItemType="Analyzer" PrivateAssets="all" />
+    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" PrivateAssets="all" OutputItemType="analyzer" ReferenceOutputAssembly="false" />
```

Sanity checking - I don't know enough to know whether this change from `Analyzer` to `analyzer` matters?

---

I don't think so... but it's not really intentional, must have gotten it while changing forth and back and testing.

```suggestion
    <ProjectReference Include="../BSATN.Codegen/BSATN.Codegen.csproj" OutputItemType="Analyzer" PrivateAssets="all" ReferenceOutputAssembly="false" />
```